hw03_Parn.py

```diff
@@ -0,0 +1,284 @@
+#################################################################################
+# Author: Din din
+# Username: Parn
+#
+# Assignment: HW03
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1sQnpXAVsiqnegxWI9VXq8K3qfozzuTXckaQppCD3Dh8/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+#
```

Comment:
Commented out code should be removed when you issue a PR. A PR should be clean, beautiful code... no garbage!